### PR TITLE
Use HTML table for assert table since markdown doesn't render properly

### PR DESCRIPTION
**Pull request by paulidale**
Trying again using a HTML table instead.

### CODE REVIEW

**Review by t8m — APPROVED** [review]

File: policies/coding-style.md [hunk 1]
```diff
@@ -742,11 +742,32 @@ For example,
...
-| -------------- | --------------- | ------------- | --------------- | ------------- |
-| assert         | not evaluated   | abort         | not evaluated   | nothing       |
-| ossl_assert    | returns 0       | abort         | returns 1       | returns 1     |
-| OPENSSL_assert | abort           | abort         | nothing         | nothing       |
+<table>
+<tr><th>Function</td>
+    <th>failure release</td>
+    <th>failure debug</td>
+    <th>success release</td>
+    <th>success debug</td>
```
Comment (t8m) [inline policies/coding-style.md:750]: `s,</td>,</th>,`

**Review by t8m — CHANGES_REQUESTED** [review]

File: policies/coding-style.md (hunk 1 above)
Comment (paulidale) [reply policies/coding-style.md]: Done

**Review by t8m — APPROVED** [review]

**Comment by mspncp** [comment]
I can't really tell the difference, except that the white-gray shading inside the table is inverted... ?

- **[before](https://github.com/openssl/technical-policies/blob/master/policies/coding-style.md#chapter-16-asserts)**
- **[after](https://github.com/openssl/technical-policies/blob/a5ef24e31632be3e75e7ed9239c3a9721bcbfd67/policies/coding-style.md#chapter-16-asserts)**

**Comment by mspncp** [comment]
> I can't really tell the difference, except that the white-gray shading inside the table is inverted... ?

If that's the only change, I'm not sure whether it is worth sacrificing the readability of the table in a simple text editor.

**Comment by mspncp** [comment]
Please forget all I said. I should have looked at https://www.openssl.org/policies/technical/coding-style.html.

**Review by mspncp — APPROVED** [review]

**Comment by paulidale** [comment]
Merged, thanks for the reviews.

**Comment by paulidale** [comment]
It's ugly but workable now.
Some further tweaking is possibly required.

**Pull request closed without merging**